content/enterprise/admin/infrastructure/demo-to-disk-migration.mdx

```diff
@@ -0,0 +1,104 @@
+---
+page_title: Demo to Mounted Disk Migration - Infrastructure Administration - Terraform Enterprise
+---
+
+# Migrating from Demo Mode to Mounted Disk Mode
+
+The demo operational mode was removed in Terraform Enterprise v202204-1 in favor of the mounted disk operational mode. If you are currently running demo mode and wish to upgrade to Terraform Enterprise v202204-1, you will need to migrate your application data from the demo mode Docker volumes to the mounted disk directory.
+
+## Migrating Application Data From Demo Mode to Mounted Disk Mode
+
+1. Connect to your Terraform Enterprise instance.
+
+1. Create the mounted disk directory. You can choose any directory you'd like.
+However, it is expected that this directory is backed by persistent storage
+that can be mounted to the instance. 
+
+  ```bash
+  mkdir -p /opt/terraform-enterprise
+  ```
+
+1. Set the `disk_path` application setting to the mounted disk directory.
+
+  ```bash
+  replicatedctl app-config set disk_path --value '/opt/terraform-enterprise'
+  ```
+
+1. Migrate the PostgreSQL data to the mounted disk directory.
+
+  ```bash
```

Comment:
just checking - a user could theoretically copy and paste these into their terminal and run them all at once, right? ir not, I wonder if we should separate them into their own separate steps.

Reply:
Yep! They can copy and paste the entire thing. I wrote it that way because I knew that's what people would try to do.